packages/app-security/src/Security.tsx

```diff
@@ -5,8 +5,12 @@ import React from "react";
 import { Provider } from "@webiny/app";
 import { SecurityProvider as ContextProvider } from "./contexts/Security";
 
-const SecurityProviderHOC = (Component: React.FC<any>): React.FC<any> => {
-    return function SecurityProvider({ children }) {
+interface SecurityProviderProps {
+    children: React.ReactNode;
+}
+
+const SecurityProviderHOC = (Component: React.ComponentType) => {
+    return function SecurityProvider({ children }: SecurityProviderProps) {
```

Comment:
Is this okay that I have started changing types for `Components` as props from `FC` to `ComponentType`?

Reply:
@brunozoric

Reply:
Ok, for now. We will see if there will be issues with React 18.